static/js/editbob.js

```diff
@@ -14,64 +18,67 @@ $(function() {
 	 * @param {string} url - A string containing the URL to tags GET request
 	 * @param {successCallback} function - creates html element tag labels
 	*/
-	$.get('/tags', function(tagArray) {
+	$.get('/api/tags', function(tagArray) {
 		$.each(tagArray, function(index, tag) {
 			$('#tag-holder').append('<label>\
 				<input type="checkbox" name="tags" value="' + tag.title + '"></input>' + tag.title + '\
 			</label><br>');
 		});
 	});
 
-	var flavors = [];
 	/**
 	 * Loads flavors data from the server using a HTTP GET request
 	 * @todo - will be deprecated soon(don't need all the flavors from db)
 	 * @param {string} url - A string containing the URL to tags GET request
 	 * @param {successCallback} function - A callback function
 	*/
-	$.get('/flavors', function(flavorArray) {
+	var flavors = [];
+	$.get('/api/flavors', function(flavorArray) {
 		// Editing Flavor of Bobs is not allowed.
-
 		flavors = flavorArray;
 		// Needs to run after flavors get filled, otherwise there is a race condition
-		fillInputFields();
+		fillInputFields(bobId);
 	});
 
 
+
+
 	/**
 	 * Populates input forms with current bob values
 	*/
-	function fillInputFields() {
+function fillInputFields(bobId) {
 
 		/**
 		 * Loads Bob Object from the server using a HTTP GET request,
 		 * @param {string} url - A string containing the route to Bob GET request
 		 * @param {successCallback} function - Fills elements with Bob data values.
 		*/
-	  $.get('/getbob?bobid=' + getUrlParameter("bobid"), function (bob) {
-			let $form = $('add-bob-form');
-
-			updateInputFromFlavor(bob.flavor, flavors);
-
-			$('#bob-id').val(bob._id);
-			$('#flavor').val(bob.flavor);
-			$('#start-date').val(bob.startDate);
-			$('#end-date').val(bob.endDate);
-
-			// fills the data forms
-			$.each(bob.data, function (key) {
-				$(String('#' + key)).val(bob.data[key]);
-			});
-
-			// fills the tag forms
-			$.each(bob.tags, function (i) {
-				$(':input[value="' + bob.tags[i] + '"]').attr('checked', true);
-			});
-	  });
-	}
+  $.get('/api/bobs/' + bobId, function (bob) {
+		cur_bob = bob;
```

Comment:
There were two spaces next to each other. I'll check if it displays better on github now